SPECS/python-setuptools_scm/python-setuptools_scm.spec

```diff
@@ -25,6 +25,7 @@ BuildRequires:  python3-pip
 BuildRequires:  cmake
 %if %{with_check}
 BuildRequires:  git
+BuildRequires:  python3-packaging >= 24.1
```

Comment:
Since you're here, could you also replace all `%if %{with_check}` with `%if 0%{?with_check}`. That's what we try to stick to in most spec. Also, the `%if %{with_check}` around the `%check` section can be removed completely.

Reply:
agreed